### ISSUE TITLE
Netflix (Widevine) not working - requests to enable protected contend ID but is already enabled

### ISSUE THREAD

**Issue by fredsco** [ungoogled-software/ungoogled-chromium#3090]
### OS/Platform

macOS

### Installed

OS/Platform's package manager

### Version

Version 130.0.6723.91 (Official Build, ungoogled-chromium) (arm64)

### Have you tested that this is not an upstream issue or an issue with your configuration?

- [X] I have tried reproducing this issue in Chrome and it could not be reproduced there
- [X] I have tried reproducing this issue in vanilla Chromium and it could not be reproduced there
- [ ] I have tried reproducing this issue in ungoogled-chromium with a new and empty profile using `--user-data-dir` command line argument and it could not be reproduced there

### Description

Netflix asks me to enable protected contend ID  but it is already on. 

### How to Reproduce?

1. Go to netflix
2. Play any content. 

### Actual behaviour

When playing any content the following is shown and the content does not play: 

>  Pardon the interruption
>  Please visit chrome://settings/content/protectedContent and make
>  sure "Sites can play protected content" is selected.
>  Netflix Home Learn More
>  Error Code M7701-1003

### Expected behaviour

Content should play

**Comment by PF4Public** [comment]
> ### Expected behaviour
> Content should play

Strictly speaking it is the opposite that is to expect. https://github.com/ungoogled-software/ungoogled-chromium/issues?q=is%3Aissue+Netflix

**Comment by fredsco** [comment]
> > ### Expected behaviour
> > Content should play
> 
> Strictly speaking it is the opposite that is to expect. https://github.com/ungoogled-software/ungoogled-chromium/issues?q=is%3Aissue+Netflix

Thank you for your comment, 

https://github.com/ungoogled-software/ungoogled-chromium/issues/1258#issuecomment-873456050 

tldr: not possible.

**Comment by lrq3000** [comment]
Also see this more technically detailed post (with the cost it would incur ~$7000 upfront and then a yearly fee): https://github.com/ungoogled-software/ungoogled-chromium/issues/1258#issuecomment-2114305561

So this will never be possible for opensource projects that are not backed by big corps.
